### ISSUE TITLE
Form elements in "Role selection" in the "Series details" dialog are not labelled

### ISSUE THREAD

**Issue by luniki** [opencast/opencast-admin-interface#598]
The form elements in the filter component:

![Screenshot 2024-06-05 at 10-04-14 Opencast](https://github.com/opencast/opencast-admin-interface/assets/2993/67a3482b-533f-4215-930c-eb203313424c)

must have labels. This could be implicit or explicit `label` elements or `aria-label` or `title` attributes.

This has critical impact on blind/low vision people.

**Comment by rlucke** [comment]
Actually, the drop-down elements have a very good accessibility implementation. However, they are not accessible with the keyboard due to incorrectly set tabindex.
